SU-SWS/stanford_components
# EARTH-294 Section Callout Blocks refactored

# READY FOR REVIEW

# Summary
- Callout Blocks & Callout Block refactored to use `pattern()` and link functions.

# Needed By (Date)
- EOS (end of spring)

# Urgency
- kinda

# Steps to Test

1. Checkout branch & EARTH-294-section-callout-blocks branch of stanford_paragraph_types
2. `drush en stanford_paragraph_callout_blocks -y`
3. Create complex page with "Callout Blocks" paragraph type (notice the plural)
4. Verify display.
5. Verify patterns/section_callout_blocks page still looks normal

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_components/pull/55

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
